content/backend/graphql-js/6-authentication.md

```diff
@@ -64,22 +64,6 @@ prisma generate
 
 </Instruction>
```

Comment:
Hi! What do you think about removing this block as well and replacing it with something like - 

>The Prisma client is generated automatically after running `prisma deploy` exposing all the CRUD methods for the newly added `User` model.